Traslate YDB concepts (#8172)
ydb-platform/ydb: ydb/docs/en/core/concepts/_includes/scan_query.md

* There is no optimization for point reads or reading small ranges of data.
* The SDK doesn't support automatic retry.

> For handling OLAP workloads in {{ ydb-short-name }}, there is a specialized type of table — [column](../datamodel/table.md#column-oriented-tables) tables. They store the data of each column separately from other columns. As a result, only those columns that are directly involved in the query are read when executing a query.

Reviewer: ```suggestion
For handling OLAP workloads in {{ ydb-short-name }}, there is a specialized type of table — [column-oriented](../datamodel/table.md#column-oriented-tables) tables. These tables store the data of each column separately from other columns. This allows only the columns directly involved in the query to be read during execution.
```
